### PR TITLE
Fix collecting SDL logs

### PR DESCRIPTION
**Pull request by VVeremjova**
Add waiting for connection for tcp socket.
Start logging right after start SDL.
Start logging only if storeFullSDLLogs flag is true

The same is in [#17](https://github.com/CustomSDL/sdl_atf/pull/17)

### CODE REVIEW

File: src/network.cc [hunk 1]
```diff
@@ -29,6 +33,17 @@ QTcpSocket *tcpSocket =
...
   tcpSocket->connectToHost(ip, port);
 
+  QTime timer;
+  timer.start();
+  while (!tcpSocket->waitForConnected()) {
+    tcpSocket->connectToHost(ip, port);
+    // Check elapsed time
+    const int time_waiting_ms = 1000;
+    if (timer.elapsed() > time_waiting_ms){
+      fprintf(stderr, "%s\n%s\n", "Error: Connection not established", strerror(errno));
```
Comment (pestOO) [inline src/network.cc]: I think error is going to be stored in `QTCPSocket::errorString` instead of `errno`.

File: src/network.cc [hunk 2]
```diff
@@ -29,6 +33,17 @@ QTcpSocket *tcpSocket =
 
   tcpSocket->connectToHost(ip, port);
 
+  QTime timer;
```
Comment (pestOO) [inline src/network.cc]: Maybe better to use `QAbstractSocket::waitForConnected`?

Comment (VVeremjova) [inline src/network.cc]: @pestOO waitForConnected has already used there, but to avoid locking on waiting I added timer

File: src/network.cc (hunk 1 above)
Comment (VVeremjova) [inline src/network.cc]: @pestOO I've checked, errno also can be used

**Pull request merged**